peripherals/gpio/gpio_l5.yaml

```
@@ -0,0 +1,67 @@
+# This GPIO is used on the STM32L5 families.
+
+"GPIO*":
+  MODER:
+    "MODE*":
+      Input: [0, "Input mode (reset state)"]
+      Output: [1, "General purpose output mode"]
+      Alternate: [2, "Alternate function mode"]
+      Analog: [3, "Analog mode"]
```

On the L5, the reset state is actually analog mode (except for the SWD/JTAG pins):

```suggestion
      Input: [0, "Input mode"]
      Output: [1, "General purpose output mode"]
      Alternate: [2, "Alternate function mode"]
      Analog: [3, "Analog mode (reset state)"]
```